docker/Dockerfile

```diff
@@ -1,24 +1,34 @@
-FROM centos:7 as stage1
+FROM ubuntu:22.04 as stage1
 
 COPY setup-hw /usr/local/bin/setup-hw
 COPY monitor-hw /usr/local/sbin/monitor-hw
 COPY collector /usr/local/sbin/collector
 COPY setup-apply-firmware /usr/local/sbin/setup-apply-firmware
 COPY setup-isoreboot /usr/local/sbin/setup-isoreboot
-
+COPY systemctl-dummy /usr/bin/systemctl
 CMD ["/usr/local/sbin/monitor-hw"]
 
-
 # Install vendor tools
 FROM stage1
 
-ARG DSU_VERSION=21.11.26
+# See https://linux.dell.com/repo/community/openmanage/
+ARG OMSA_VERSION=11000
+ARG UBUNTU_VERSION=jammy
 
-RUN curl -sf -o bootstrap.cgi https://linux.dell.com/repo/hardware/DSU_${DSU_VERSION}/bootstrap.cgi \
-    && bash bootstrap.cgi \
-    && rm bootstrap.cgi \
-    && yum update -y \
-    && yum install -y syscfg srvadmin-idracadm7 dmidecode \
-    && yum clean all
+# Setup DELL repository
+RUN apt update -y && apt-get install wget pgp apt-utils -y \
```

Comment:
Is `pgp` necessary?
`apt-utils` depends on GnuPG for itself.

Reply:
I can remove it if we not do pgp verify. This procedure is inherit from OMSA installation document.
In generally, if trusted site, It no need verify in Dockerfile I think. 
so, I will remove doing pgp verify, and related module.

Reply:
We can not remove pgp due to it is called from apt-add command.